tests/pkg_tomsfastmath/tests/01-run.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python3
+
+# Copyright (C) 2016 Kaspar Schleiser <kaspar@schleiser.de>
+# Copyright (C) 2016 Mathias Tausig <mathias.tausig@fh-campuswien.ac.at>
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+import os
+import sys
+
+sys.path.append(os.path.join(os.environ['RIOTBASE'], 'dist/tools/testrunner'))
+import testrunner
+
+def testfunc(child):
+    child.expect('All tests OK.')
+
+if __name__ == "__main__":
+    sys.exit(testrunner.run(testfunc))
```

Comment:
The default timeout for the testrunner if 10 seconds. On the samr21-xpro the test takes somewhere between 12 and 13 seconds, so I think we should raise the timeout here to maybe 20 or 30?